flink-runtime/src/main/scala/org/apache/flink/runtime/minicluster/FlinkMiniCluster.scala

```diff
@@ -51,10 +51,13 @@ import scala.concurrent.{ExecutionContext, Future, Await}
 abstract class FlinkMiniCluster(
     val userConfiguration: Configuration,
     val singleActorSystem: Boolean,
+    val synchronousDispatcher: Boolean,
     val streamingMode: StreamingMode) {
 
-  def this(userConfiguration: Configuration, singleActorSystem: Boolean) 
-         = this(userConfiguration, singleActorSystem, StreamingMode.BATCH_ONLY)
+  def this(userConfiguration: Configuration,
+           singleActorSystem: Boolean,
+           synchronousDispatcher: Boolean)
+       = this(userConfiguration, singleActorSystem, synchronousDispatcher, StreamingMode.BATCH_ONLY)
```

Comment:
I think the field `synchronousDispatcher` should not be part of the `FlinkMiniCluster`. The field seems to be only used for testing, referenced in `TestingCluster`, but the `FlinkMiniCluster` is also used to execute Flink programs locally.